quickstarts/microsoft.machinelearningservices/machine-learning-service-create-aks/metadata.json

```diff
@@ -8,9 +8,5 @@
   "dateUpdated": "2021-05-27",
   "environments": [
     "AzureCloud"
-  ],
-  "tags.subscriptiontype": "Azure4Student",
-  "tags.resource": "Machine Learning Services",
-  "tags.level": "beginner",
-  "tags.cost": "$$"
+  ]
```

Comment:
these tags need to be included as an object property, e.g.

```
"tags": {
  "subscriptionType": "Azure4Student"
  ...
}
```

Reply:
Added the tags accordingly.